src/pageServices/contentScripts/highlight.js

```diff
@@ -342,19 +342,11 @@ export const highlightOnPage = () => {
   const onElementDblClick = (evt) => {
     const locatorId = evt.target.id || evt.target.offsetParent.id;
     const element = predictedElements?.find((elem) => elem.jdnHash === locatorId);
-    const isElementAddedToPO = element.generate;
 
     sendMessage({
-      message: "ELEMENT_UNSET_ACTIVE",
```

Comment:
please, remove this from `scriptMessageHandler.ts` as well

Reply:
done